diff.py

```diff
@@ -0,0 +1 @@
+tools/asm-differ/diff.py
```

Comment:
I'm not completely sure we want to clutter up the main directory. The reason for the ./tools/ directory is to put stuff like that there.
I guess we did used to have a diff.sh there.

Reply:
This file is actually a symlink (like a Windows's shortcut). I don't know why Github shows it like this.

Reply:
Also, the problem with `diff.sh` is it doesn't allow to change the arguments passed to `diff.py` without modifying the script.

Reply:
I know. I think this is fine.